content/docs/100-get-started/200-build-an-app.mdx

```diff
@@ -3,6 +3,95 @@ title: Build an App
 excerpt: A quick tutorial that covers the basics of building your first app.
 ---
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+This tutorial will walk you through the basics of building a Raydiant app. We’ll use the [Simple Weather](https://simple-weather-raydiant-app.vercel.app/?location=New%20York) app to set up its inputs and preview the app in the Raydiant Dashboard.
 
-<Callout>Sample callout</Callout>
+You won’t need to write any code but there are instructions for deploying the weather app at the end of the tutorial if you’d like to use it as a starting point for building your own app.
+
+## 1. Before We Start
+
+Before you can create your first app you will need to [create an account](https://www.notion.so/Create-an-Account-dbf9280fa5444102b0b4cea90ce8307c) to access the Developer Portal and the Raydiant Dashboard.
+
+Once you have an account, sign in to the [Developer Portal](https://developers.raydiant.com) to begin the tutorial.
+
+## 2. Create the App
+
+From the [Apps](https://developers.raydiant.com/apps) section, we’ll start by creating the Simple Weather app and adding an icon and thumbnail.
+
+1. Click the “Create app” button [TODO: add icon].
+2. Set the name of the app to “Simple Weather”.
+3. Set the description of the app to “Display local weather from around the world.”
+4. Set the app ID to something unique like `alices-simple-weather`
+5. Upload this <a href="/simple-weather-thumbnail.jpg" download>thumbnail</a> and <a href="/simple-weather-icon.jpg" download>icon</a> to the “Assets” section.
+
+## 3. Add a Location Input
+
+Next, let’s set up your app’s URL template and a location input to allow users to type in their city.
+
+```
+ https://simple-weather-raydiant-app.vercel.app?location=:location&units=:units
+```
```

Comment:
This seems to cause the page to overflow. The desired behaviour is that the code block has a horizontal scroll bar. Looking at the CSS it seems like it should but for some reason it's not.